src/sentry/api/endpoints/organization_discover.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+
+from rest_framework.response import Response
+from sentry.api.bases.organization import OrganizationPermission
+from sentry.api.bases import OrganizationMemberEndpoint
+
+from sentry.models import Project, ProjectStatus, OrganizationMemberTeam
+
+from sentry import roles
+
+
+from sentry.utils import snuba
+
+
+class OrganizationDiscoverPermission(OrganizationPermission):
+    scope_map = {
+        'POST': ['org:read', 'project:read']
+    }
+
+
+class OrganizationDiscoverEndpoint(OrganizationMemberEndpoint):
+    permission_classes = (OrganizationDiscoverPermission, )
+
+    def do_query(self, start, end, selected_columns=None, conditions=None, having=None, filters=None,
```

Comment:
given this is just a passthrough to raw_query, maybe its better to just use `**kwargs` here instead of enumerating them and redoing all the defaults.